### PR TITLE
[13.0][IMP][mis_builder_operating_unit] Filter account entries not having operating unit

### PR DESCRIPTION
**Pull request by robyf70**
Add explict filter for account entries without having set the operating unit"

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
IMO all journal items should have an operating unit.

**Comment by robyf70** [comment]
> IMO all journal items should have an operating unit.

As I know you can still post account entries without OU, so it isn't mandatory field

**Comment by AaronHForgeFlow** [comment]
Yes, that is a good point. I will wait for others feedback.

**Review by AaronHForgeFlow — APPROVED** [review]
Code LG

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
I think the filter show be added in the main instance as well. If you are not in comparison mode users should be able to filter journals by no operating unit. Can you check this? https://github.com/LevelPrime/operating-unit/pull/1

**Comment by robyf70** [comment]
> I think the filter show be added in the main instance as well. If you are not in comparison mode users should be able to filter journals by no operating unit. Can you check this? [LevelPrime#1](https://github.com/LevelPrime/operating-unit/pull/1)

You are right! I will add this feature on the main instance as well

**Comment by robyf70** [comment]
@AaronHForgeFlow I've just pushed the requested changes

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM thanks

**Comment by robyf70** [comment]
@AaronHForgeFlow Do we need another review to be able to merge?

File: mis_builder_operating_unit/model/mis_builder.py
Comment (MiquelRForgeFlow) [inline mis_builder_operating_unit/model/mis_builder.py]:
```suggestion
        elif sudoself.report_instance_id.has_no_operating_unit:
            aml_domain.append(("operating_unit_id", "=", False,))
```

File: mis_builder_operating_unit/model/mis_builder.py
Comment (MiquelRForgeFlow) [inline mis_builder_operating_unit/model/mis_builder.py]:
```suggestion
        else:
            if sudoself.report_instance_id.operating_unit_ids:
```

File: mis_builder_operating_unit/model/mis_builder.py
Comment (robyf70) [reply mis_builder_operating_unit/model/mis_builder.py]: Ok! Done!

File: mis_builder_operating_unit/model/mis_builder.py
Comment (robyf70) [reply mis_builder_operating_unit/model/mis_builder.py]: Ok! Done!

**Comment by robyf70** [comment]
@MiquelRForgeFlow Are you able to review the last changes so we can finally get it merged?

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**
